packages/ui/src/common/components/Tooltip/Tooltip.tsx

```diff
@@ -121,11 +121,16 @@ export const Tooltip = ({
               >
                 {tooltipTitle && <TooltipPopupTitle>{tooltipTitle}</TooltipPopupTitle>}
                 <TooltipText>{tooltipText}</TooltipText>
-                {tooltipLinkURL && (
+                {tooltipLinkURL && (tooltipLinkURL.startsWith('#') || tooltipLinkURL.startsWith('/')) ? (
```

Comment:
```suggestion
                {tooltipLinkURL && (isExternalLink(tooltipLinkUrl)) ? (
```

To make this condition a bit easier to read I would extract this to a helper function.

I think that there should be additional bracket around both components. ATM if `tooltipLinkURL` is falsy the `<TooltipExternalLink />`  will be rendered.

I think it should be something like:

```tsx
{tooltipLinkUrl && (
  isExternalLink(tooltipLinkURL) ? (
   <TooltipLink /> 
  ) : (
   <TooltipExternalLink /> 
  )
)} // <- two closing brackets
```

Reply:
Makes sense, pushed the update.